app/brave_settings_strings.grdp

```diff
@@ -807,6 +807,16 @@
   <message name="IDS_SETTINGS_SITE_SETTINGS_SOLANA_BLOCK" desc="Label for solana site settings.">
     Block sites from accessing the Solana provider API
   </message>
+  <!-- Cardano -->
+  <message name="IDS_SETTINGS_SITE_SETTINGS_CARDANO" desc="Label for cardano site settings.">
+    Cardano
+  </message>
+  <message name="IDS_SETTINGS_SITE_SETTINGS_CARDANO_ASK" desc="Label for cardano site settings.">
+    Ask when a site wants to access the Cardano provider API
+  </message>
+  <message name="IDS_SETTINGS_SITE_SETTINGS_CARDANO_BLOCK" desc="Label for cardano site settings.">
+    Block sites from accessing the Cardano provider API
+  </message>
```

Comment:
Do we need to say 'provider' in these 2 messages? Seems like `Cardano API` would suffice?

Reply:
This is done in a similar way to other coins